src/test/java/org/sonar/plugins/github/PullRequestIssuePostJobTest.java

```diff
@@ -167,6 +173,99 @@ public void testPullRequestAnalysisWithNewIssues() {
+    when(newIssue2.message()).thenReturn("msg3");
+
+    // Critical => Should be displayed in 5th position
+    Issue newIssue3 = mock(Issue.class);
+    when(cache.byKey("foo:src/Foo.php")).thenReturn(inputFile1);
+    when(newIssue3.componentKey()).thenReturn("foo:src/Foo.php");
+    when(newIssue3.line()).thenReturn(3);
+    when(newIssue3.ruleKey()).thenReturn(RuleKey.of("repo", "rule"));
+    when(newIssue3.severity()).thenReturn(Severity.CRITICAL);
+    when(newIssue3.isNew()).thenReturn(true);
+    when(newIssue3.message()).thenReturn("msg4");
+
+    // Critical => Should be displayed in 7th position
+    Issue newIssue4 = mock(Issue.class);
+    when(cache.byKey("foo:src/Foo.php")).thenReturn(inputFile1);
+    when(newIssue4.componentKey()).thenReturn("foo:src/Foo.php");
+    when(newIssue3.line()).thenReturn(13);
+    when(newIssue4.ruleKey()).thenReturn(RuleKey.of("repo", "rule"));
+    when(newIssue4.severity()).thenReturn(Severity.INFO);
+    when(newIssue4.isNew()).thenReturn(true);
+    when(newIssue4.message()).thenReturn("msg5");
+
+    // Blocker on project => Should be displayed 1st position
+    Issue issueOnProject = mock(Issue.class);
+    when(issueOnProject.ruleKey()).thenReturn(RuleKey.of("repo", "rule"));
+    when(issueOnProject.componentKey()).thenReturn("foo");
+    when(issueOnProject.severity()).thenReturn(Severity.BLOCKER);
+    when(issueOnProject.isNew()).thenReturn(true);
+    when(issueOnProject.message()).thenReturn("msg6");
+
+    // Blocker and no line => Should be displayed in 2nd position
+    Issue globalIssue = mock(Issue.class);
+    when(cache.byKey("foo:src/Foo.php")).thenReturn(inputFile1);
+    when(globalIssue.componentKey()).thenReturn("foo:src/Foo.php");
+    when(globalIssue.line()).thenReturn(null);
+    when(globalIssue.ruleKey()).thenReturn(RuleKey.of("repo", "rule"));
+    when(globalIssue.severity()).thenReturn(Severity.BLOCKER);
+    when(globalIssue.isNew()).thenReturn(true);
+    when(globalIssue.message()).thenReturn("msg7");
+
+    when(issues.issues()).thenReturn(Arrays.<Issue>asList(newIssue, globalIssue, issueOnProject, newIssue4, newIssue2, issueInSecondFile, newIssue3));
+    when(pullRequestFacade.hasFile(any(InputFile.class))).thenReturn(true);
+    when(pullRequestFacade.hasFileLine(any(InputFile.class), anyInt())).thenReturn(false);
+
+    pullRequestIssuePostJob.executeOn(null, null);
+
+    verify(pullRequestFacade).addGlobalComment(commentCaptor.capture());
+
+    String comment = commentCaptor.getValue();
+    assertThat(comment.indexOf("msg6")).isLessThan(comment.indexOf("msg7"));
+    assertThat(comment.indexOf("msg7")).isLessThan(comment.indexOf("msg1"));
+    assertThat(comment.indexOf("msg1")).isLessThan(comment.indexOf("msg2"));
+    assertThat(comment.indexOf("msg2")).isLessThan(comment.indexOf("msg4"));
+    assertThat(comment.indexOf("msg4")).isLessThan(comment.indexOf("msg3"));
+    assertThat(comment.indexOf("msg3")).isLessThan(comment.indexOf("msg5"));
```

Comment:
Could be changed to: assertThat(comment).containsSequence("msg6", "msg7", "msg1", "msg2", "msg4", "msg3")